charts/k8s-monitoring/templates/agent_config/_pod_logs_discovery.river.txt

```diff
@@ -35,7 +35,7 @@ discovery.relabel "pod_logs" {
     target_label = "container"
   }
   rule {
-    source_labels = ["__meta_kubernetes_namespace", "__meta_kubernetes_pod_name"]
+    source_labels = ["__meta_kubernetes_namespace", "__meta_kubernetes_pod_container_name"]
```

Comment:
@mar4uk Note that we're changing to job label to `<namespace>/<container>` in this change as well.

Reply:
Note to @grafana/k8s-monitoring-dev , too. We don't use the `job` label for pod logs, other than to assert that they *dont* belong to cluster events, so this should be a safe change.

@BeverlyJaneJ, I don't think any of our docs mention the `job` label for pod logs, but it might be worth a quick search.

Reply:
Not that I can find.